htlcswitch/hop/iterator.go

```diff
@@ -169,11 +169,18 @@ func calculateForwardingAmount(incomingAmount lnwire.MilliSatoshi, baseFee,
 // tests dependent from the sphinx internal parts.
 type OnionProcessor struct {
 	router *sphinx.Router
+
+	// Adds the option to disable forwarding payments in blinded routes
+	// by failing back any blinding-related payloads as if they were
+	// invalid.
+	disallowRouteBlinding bool
```

Comment:
Can this option live within the switch rather than the onion processor? This way when the link gets an HTLC that wants to do route blinding, it just bails there, rather than choking when it tries to process the HTLC.

Reply:
Looks like was moved into a link level config option in the latest diff 👍